source/core/ut_annotations.pkb

```diff
@@ -30,7 +30,7 @@ create or replace package body ut_annotations as
   c_rgexp_identifier            constant varchar2(50) := '[a-z][a-z0-9#_$]*';
   c_annotation_block_pattern    constant varchar2(200) := '(({COMMENT#.+}'||chr(10)||')+)( |'||chr(09)||')*(procedure|function)\s+(' ||
                                                            c_rgexp_identifier || ')';
-  c_annotation_pattern          constant varchar2(50) := gc_annotation_qualifier || c_rgexp_identifier || '(\(.*?\)$)?';
+  c_annotation_pattern          constant varchar2(50) := gc_annotation_qualifier || c_rgexp_identifier || '[ '||chr(9)||']*(\(.*?\)$)?';
```

Comment:
Can you make the regexp work with trailing spaces?
'[ '||chr(9)||]*(\(.*?\)[ '||chr(9)||']*$)?'